### PR TITLE
feat: launch blog

### PR DESCRIPTION
**Pull request by daniwk**

### CODE REVIEW

File: blog.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+## (Optional) View and manage cluster with k9s
+`k9s` offers a bit more user-friendly interface than the standard `kubectl` CLI to your Kubernetes cluster.
+- Install k9s: `brew install derailed/k9s/k9s`
+- Connect to your newly created cluster: `k9s`
+
+[screenshot of successfully connected cluster]
+
+## Tilt
+Tilt deserves a blog post in its own. It's a toolkit which greatly improves the local development of Kubernetes apps. It provides a tight feedback loop by automatically building your Dockerfile and deploying it to your local k8s cluster live with every code edit, in addition to a whole lot more. We'll use it here to skip the docker build and push process.
```
Comment (stianfro) [inline blog.md]: Suggest changing from:
```
Tilt deserves a blog post in its own
```
To:
```
Tilt deserves its own blog post.
```

Or:
```
Tilt really deserves its own blog post.
```

File: blog.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+3. Use `k9s` to watch KEDA scale out the receiver app (find `sb-queue-receiver` under deployments by using `:deploy` and watch the READY column).
+
+[screenshot/gif]
+
+## Clean up resources
+1. Delete kind by running: `ctlptl delete cluster kind`
+2. Delete Azure resource group by opening up a Cloud Shell in the Azure Portal and run: `az group delete -n rg-autoscale-dev`
+
+## Summary
+We have now seen how we can use KEDA to autoscale our Kubernetes apps based on external data source, such as number of messages in a Service Bus Queue. Once you have your k8s cluster and external data source available and ready, it's actually straightforward to implement KEDA. However, after having implemented it and running in production for a while, we want to share our learnings.
```
Comment (stianfro) [inline blog.md]: based on **an** external data source

File: blog.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+KEDA delivers on its promise to be a single-purpose and lightweight component. As we have seen, from the developers viewpoint, implementation of autoscaling is simple. Additionally, KEDA offers a large number of ready-made [scalers](https://keda.sh/docs/2.9/scalers/).
+
+But you should plan ahead with your platform team before rolling it out in production. First off, it's important to define resource request and limits for your workloads. This helps the Kubernetes scheduler schedule your pods without [bringing down your production environment](https://engineering.intility.com/article/guide-to-high-availability-in-kubernetes#resources-and-scheduling). Secondly, if your workloads need to scale out to 50+ replicas you should consider setting up a dedicated node(s) to separate more stable production workloads from the workloads with surges. Implementing autoscaling on your k8s cluster could also be good idea.
```
Comment (stianfro) [inline blog.md]:
```
important to define resource requests and limits
                                    ^
```

File: blog.md (hunk 1 above)
Comment (daniwk) [reply blog.md]: `Tilt really deserves its own blog post.` it is.

**Pull request merged**